### PR TITLE
Polkadot 0.9.13

### LINKED ISSUE #54: upgrade to polkadot 0.9.13
https://github.com/paritytech/cumulus/releases/tag/polkadot-v0.9.13

### PR DESCRIPTION
**Pull request by brenzi**
closes #54 

TODO before merge:
* [x] bump crate versions and spec_version
* DON't: release new hard-coded specs, (for shell, based on results of #61)

### CODE REVIEW

**Comment by brenzi** [comment]
clippy doesn't like it, but mostly complains about upstream code. Do we really want to fix this?

**Comment by brenzi** [comment]
tried to launch the parachain as follows with collator 6765e9bead5e517c8825e6275bab7ed7a8493675:

1. launch relaychain `node ../polkadot-launch/dist/cli.js polkadot-launch/launch-rococo-local.json`
2. register paraid 2015 and parathread with the files attached above
3. upgrade parathread to parachain
4. run one collator `./target/release/integritee-collator         --collator         --chain shell-rococo         --rpc-cors all         --name integritee-rococo-collator-1         -- --execution wasm --chain rococo-local-raw.json `
5. inject first of five authority keys

The parachain won't produce blocks, but no error msg:

```
2022-01-14 09:03:54 [Relaychain] ✨ Imported #174 (0xf7a2…938f)    
2022-01-14 09:03:54 [Parachain] Starting collation. relay_parent=0xf7a2a2371b65c82f9ac355bda7548eb980f9c5cfdbb2873ddfa3b6d2dbfe938f at=0x724c168d8e86b78b831c641e2cc822b8d1bf99fa0b4b28fe59985cd6fd580215
2022-01-14 09:03:56 [Parachain] 💤 Idle (0 peers), best: #0 (0x724c…0215), finalized #0 (0x724c…0215), ⬇ 0 ⬆ 0    
2022-01-14 09:03:56 [Relaychain] 💤 Idle (3 peers), best: #174 (0xf7a2…938f), finalized #171 (0xaf4c…1e2d), ⬇ 1.7kiB/s ⬆ 1.4kiB/s    
2022-01-14 09:04:00 [Relaychain] ✨ Imported #175 (0xb689…b1a5)    
2022-01-14 09:04:00 [Parachain] Starting collation. relay_parent=0xb689719e01f63979d95f29ca4743b82feb7fb1ea8f42b92f5a3e725340dbb1a5 at=0x724c168d8e86b78b831c641e2cc822b8d1bf99fa0b4b28fe59985cd6fd580215
2022-01-14 09:04:01 [Parachain] 💤 Idle (0 peers), best: #0 (0x724c…0215), finalized #0 (0x724c…0215), ⬇ 0 ⬆ 0    
2022-01-14 09:04:01 [Relaychain] 💤 Idle (3 peers), best: #175 (0xb689…b1a5), finalized #172 (0x3be4…8bc2), ⬇ 1.4kiB/s ⬆ 1.0kiB/s    
2022-01-14 09:04:06 [Relaychain] ✨ Imported #176 (0xdbf5…6780)    
2022-01-14 09:04:06 [Parachain] Starting collation. relay_parent=0xdbf587996c129fc73d279e38d3852c00e5f83dbda8a6e1d5bbf0f5e194506780 at=0x724c168d8e86b78b831c641e2cc822b8d1bf99fa0b4b28fe59985cd6fd580215
2022-01-14 09:04:06 [Parachain] 💤 Idle (0 peers), best: #0 (0x724c…0215), finalized #0 (0x724c…0215), ⬇ 0 ⬆ 0    
2022-01-14 09:04:06 [Relaychain] 💤 Idle (3 peers), best: #176 (0xdbf5…6780), finalized #173 (0x449f…dc2f), ⬇ 1.8kiB/s ⬆ 1.5kiB/s    
2022-01-14 09:04:11 [Parachain] 💤 Idle (0 peers), best: #0 (0x724c…0215), finalized #0 (0x724c…0215), ⬇ 0 ⬆ 0    
2022-01-14 09:04:11 [Relaychain] 💤 Idle (3 peers), best: #176 (0xdbf5…6780), finalized #174 (0xf7a2…938f), ⬇ 1.0kiB/s ⬆ 0.9kiB/s    
2022-01-14 09:04:12 [Relaychain] ✨ Imported #177 (0x46da…c57c)    
2022-01-14 09:04:12 [Parachain] Starting collation. relay_parent=0x46da57aa3b609cbb8097ac333f8d253429091691f7b3b2e44dc23b51ad8fc57c at=0x724c168d8e86b78b831c641e2cc822b8d1bf99fa0b4b28fe59985cd6fd580215
2022-01-14 09:04:16 [Parachain] 💤 Idle (0 peers), best: #0 (0x724c…0215), finalized #0 (0x724c…0215), ⬇ 0 ⬆ 0    
2022-01-14 09:04:16 [Relaychain] 💤 Idle (3 peers), best: #177 (0x46da…c57c), finalized #175 (0xb689…b1a5), ⬇ 1.7kiB/s ⬆ 1.3kiB/s    

```

File: polkadot-parachains/src/cli.rs [hunk 1]
```diff
@@ -1,8 +1,7 @@
-// Copyright 2021 Integritee AG and Supercomputing Systems AG
-// This file is part of the "Integritee parachain" and is
-// based on Cumulus from Parity Technologies (UK) Ltd.
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
 
-// Integritee parachain is free software: you can redistribute it and/or modify
+// Cumulus is free software: you can redistribute it and/or modify
```
Comment (clangenb) [inline polkadot-parachains/src/cli.rs:4]: Makes sense to me, still wondering if this is intentional?

File: polkadot-parachains/src/service.rs [hunk 2]
```diff
@@ -379,46 +415,269 @@ pub fn rococo_parachain_build_import_queue(
...
+		} else {
+			self.relay_chain_verifier.verify(block_import).await
+		}
+	}
 }
 
+/// Generic import queue for the integritee- and shell parachain runtime.
+///
+/// The trait bounds of the generic parameters are copied from the above `start_node_impl` except
+/// for the one mentioned below.
```
Comment (clangenb) [inline polkadot-parachains/src/service.rs:574]: `except for the one mentioned below`  is obsolete, if it still exists in the encointer-parachain, it's wrong. It's now identical to the `start_node_impl`.

File: polkadot-parachains/src/service.rs [hunk 3]
```diff
@@ -427,17 +686,71 @@ pub async fn start_rococo_parachain_node(
...
+		shell_parachain_build_import_queue,
+	)
+	.await
+}
+
+/// Generic implementation introduced by integritee.
+///
+/// The trait bounds of the generic parameters are copied from the above `start_node_impl` except
+/// for the one mentioned below.
+pub async fn start_parachain_node<RuntimeApi, Executor, BIQ>(
```
Comment (clangenb) [inline polkadot-parachains/src/service.rs:707]: same as with the above comment.

**Review by clangenb — COMMENTED** [review]
Code looks good to me.

File: polkadot-parachains/src/cli.rs (hunk 1 above)
Comment (brenzi) [reply polkadot-parachains/src/cli.rs:4]: not intentional. but I'll leave it as it's copy-paste

**Pull request merged**